server/server_test.go

```diff
@@ -23,6 +23,11 @@ var (
 	csrv         *httptest.Server
 	clientID     = "111111"
 	clientSecret = "11111111"
+
+	plainChallenge = "plaintest"
```

Comment:
BTW this is too short and will fail the length validation (under 43 characters long).

Ex: [see this](https://github.com/go-fed/oauth2/commit/645a05c52a2a35335781e5bf84f7181ac463e90b).

Reply:
Hi, thanks for the review. Nice catch. Fixed.